Lombiq.HelpfulLibraries.Core/Lombiq.HelpfulLibraries.Core.csproj

```diff
@@ -5,6 +5,13 @@
     <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
A project nevét nem kell Core-ra változtatni, az maradjon, nincs szükség ennyire breaking change-re (meg amúgy akkor a namespace-eket is kellene változtatni, ami főleg fölösleges).

Reply:
Oof ezt én kértem, de jogos. A mappa neve maradhat, ugye?

Reply:
Annak se igazán látom értelmét, hogy más legyen. Kicsit lehet fura, hogy van egy sima HelpfulLibraries, meg HelpfulLibraries.RestEase, de jó úgy szeirntem.

Reply:
Nekem meg az nem tetszik, hogy a path akkor `/Libraries/Lombiq.HelpfulLibraries/Lombiq.HelpfulLibraries/` lenne 😵.

Reply:
Gyönyörű path 😍. Tényleg kicsit hülyén néz ki, de szerintem nem érdemes rajta sokat agyalni. Pont így amúgy se nagyon fogja senki látni, mert GH-on nem így jelenik meg, ahogy a solution explorerben sem, a fájlrendszert pedig így azért ritkán tallózzuk.